### PR TITLE
fix(CX-1945): issue with network error message

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

[This is blocks users from using Eigen properly, if it looks good to you please merge it]

This PR resolves [CX-1945]

### Description
- Fix issue with network error message

I removed the network error message from the app providers and only added it the new onboarding since it's the only place where we don't have the "Unable to load" screen
https://user-images.githubusercontent.com/11945712/134346878-3811f65e-1796-4bf0-a8dd-d9f7b56d0ef4.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

- Add No Internet connection modal - mounir

#### Dev changes

-

</details>

[CX-1945]: https://artsyproduct.atlassian.net/browse/CX-1945

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Android user-facing changes (Add No Internet connection modal - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 435436c572539a6aa5faf72be66a445b461fed32
</p>

File: src/lib/utils/__tests__/NetworkAwareProvider-tests.tsx [hunk 1]
```diff
@@ -1,38 +0,0 @@
-import { useNetInfo } from "@react-native-community/netinfo"
-import { Text } from "palette"
-import React from "react"
-import { Modal } from "react-native"
-import ReactTestRenderer from "react-test-renderer"
-import { NetworkAwareProvider } from "../NetworkAwareProvider"
-
-describe("NetworkAwareProvider", () => {
```
Comment (MounirDhahri) [inline src/lib/utils/__tests__/NetworkAwareProvider-tests.tsx:8]: no idea how to test emitting events 😅

**Review by olerichter00 — COMMENTED** [review]
Looks good to me!

What do you think about putting the `NetworkAwareProvider` behind a feature flag to be able to disable it with echo in the case this doesn't work as expected?

**Comment by brainbicycle** [comment]
Agree with Ole's suggestion we should have a feature flag for this! @MounirDhahri can you add?

**Comment by MounirDhahri** [comment]
Of course I will 🙌

**Pull request merged**